Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -1151,6 +1151,7 @@ void BrowserWindow::resizeEvent(QResizeEvent* event)
     QWidget::resizeEvent(event);
 
     for_each_tab([&](auto& tab) {
+        tab.view().set_viewport_size({ static_cast<int>(tab.view().width() * m_device_pixel_ratio), static_cast<int>(tab.view().height() * m_device_pixel_ratio) });
```

Comment:
commit description mentioned programmatic resize request (on_resize_window), but this event is triggered when window is resized by a user interactive, no? probably would be better to have another commit that describes why this change is needed.